cas-server-documentation/installation/SPNEGO-Authentication.md

```diff
@@ -32,7 +32,7 @@ SPNEGO support is enabled by including the following dependency in the Maven WAR
 ```xml
 <dependency>
   <groupId>org.jasig.cas</groupId>
-  <artifactId>cas-server-support-spnego-webflow</artifactId>
```

Comment:
This is not right the change. The webflow module should be creating everything SWF needs. If it's not, we should look there and see what might be failing.